docs/data-cookbooks/sips-workflow.md

```diff
@@ -0,0 +1,121 @@
+# Science Investigator-led Processing Systems (SIPS)
+The Cumulus ingest workflow supports the SIPS workflow. In the following document, we'll discuss what a SIPS workflow is and how to set one up in a Cumulus instance.
+
+In this document, we assume the user already has a provider endpoint configured and ready with some data to ingest. We'll be using an S3 provider and ingesting from a MOD09GQ collection.
+
+
+## Setup
+
+1. We need to build a collection. Details on collections can be found [here](./setup.md#collections). The following collection will have `MOD09GQ` as a collection name, `006` as a version, and is configured to pull pdrs from `${bucket}/cumulus-test-data/pdrs` in S3 (where `${bucket}` is configured in the provider).
```

Comment:
I think headers would be a little easier to read than the numbering here. Also I'm thinking maybe provider should go first since we reference the provider here. Provider is using the cumulus-test-internal bucket, but here we're using cumulus-test-data, which I think is confusing. Can we have `cumulus-test-data` for both?